app/src/main/java/com/example/mpweather/MainActivity.kt

```diff
@@ -28,21 +29,18 @@ class MainActivity : AppCompatActivity() {
     private val appid = "62985350695c6c5b6f476b823778d5e5"
 
     // Display tempiture in two decimal format design
-    var df = DecimalFormat("#.##")
+    internal var decimalFormatting = DecimalFormat("#.##")
```

Comment:
I made this variable internal and I also made it have better naming convention so that it is easier understand and is more descriptive and because only this module uses this variable. Here are references if you would like to look further into this.
https://towardsdatascience.com/data-scientists-your-variable-names-are-awful-heres-how-to-fix-them-89053d2855be. 
https://kotlinlang.org/docs/visibility-modifiers.html#classes-and-interfaces